app/controllers/donations_controller.rb

```diff
@@ -154,4 +155,8 @@ def total_value(donations)
     end
     total_value_all_donations
   end
+
+  def total_money_raised(donations)
+    donations.inject(0) { |sum, d| sum + d.money_raised.to_i }
```

Comment:
I'm not sure if we want to cast to integer for each sub-part of the sum. Could this maybe be done as something like `donations.sum(&:money_raised)` ?

Reply:
@mdworken  I've refactored this to use the `sum` method, which is better. However, I have to call to_i because money_raised could be nil.